Documentation/Preparator.txt

```diff
@@ -0,0 +1 @@
+Prepare simply passes the ratings from TrainingData to PreparedData
```

Comment:
instead of having a documentation folder I think it would make more sense to consolidate all the information in a markdown